final_prep/README.md

```diff
@@ -23,52 +23,52 @@ Some things to consider as you refactor include...
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [X] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+-  <img width="1333" alt="Screen Shot 2022-02-13 at 12 45 56 AM" src="https://user-images.githubusercontent.com/98673086/153744214-923961b7-8cac-4206-a3e5-0b1d76d96d0d.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 08 AM" src="https://user-images.githubusercontent.com/98673086/153744218-ebf322a0-a074-468b-aa58-1f359a4c26e8.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 22 AM" src="https://user-images.githubusercontent.com/98673086/153744227-267473c6-5122-48ec-98e8-8f88bd9f0f3d.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] [link](https://gist.github.com/lcole37/a942e8c1e83f105e0fa59748fab48824)
```

Comment:
This might be a good area to spend more time before Mod 1 starts. Nothing wrong with what you have—great start. I'd encourage you to reflect on some of the questions and work on your 'introduction' at the end. More context will help your mentor jump right into building a strong relationship.